onnxscript/rewriter/ort_fusions/sdpa_test.py

```diff
@@ -22,7 +23,9 @@
 S = 8  # sequence length
 H = 128  # head size
 SCALE_FACTOR = math.sqrt(H)
+MUL_SCALE_FACTOR = 1.0 / SCALE_FACTOR
 SQRT_SCALE_FACTOR = math.sqrt(SCALE_FACTOR)
+MUL_SQRT_SCALE_FACTOR = math.sqrt(MUL_SCALE_FACTOR)
```

Comment:
minor nit: SQRT_MUL_SCALE_FACTOR may be a slightly better name (as shorthand for SQRT of MUL_SCALE_FACTOR).